src/component/todoList.js

```diff
@@ -0,0 +1,68 @@
+export default function TodoList(app) {
+	this.$todoList = document.querySelector("#todo-list");
+	this.todoItems = [];
+
+  this.setState = updatedTodoItems => {
+    this.todoItems = updatedTodoItems;
+    this.render(this.todoItems);
+  };
+
+  this.render = items => {
+    const template = items.map(todoItemTemplate);
+    this.$todoList.innerHTML = template.join("");
+  };
+
+	const todoItemTemplate = (item) => {
+    return `<li class=${item.status}>
+    <div class="view">
+      <input class="toggle" type="checkbox" ${item.status === "completed" ? "checked": ""}/>
+      <label class="label">${item.text}</label>
+      <button class="destroy"></button>
+    </div>
+    <input type="hidden" id="item-id" value="${item.id}"/>
+    <input class="edit" value="${item.text}" />
+  </li>`;
+	};
+
+	this.filterItems = status => {
+		if (status !== "all") {
+			this.todoItems = app.todoItems.filter(item => item.status === status);
+			this.render(this.todoItems);
+		}
+		else {
+			this.render(app.todoItems);
+		}
+	}
+
+	const onClick = event => {
+		if (event.target.className === "destroy") {
+			// 여기선 id만 추출해서 todoApp의 delete 함수 호출시 인자로 넘김
+			const itemId = parseInt(event.target.closest("li").querySelector("#item-id").value);
```

Comment:
```suggestion
			const itemId = +event.target.closest("li").querySelector("#item-id").value;
```
처럼 `+` 를 이용해서 숫자값을 가져오는 방법도 있습니다. 다만 조금의 차이는 있으니 [링크 한번 보시면 좋을거같습니다.](https://stackoverflow.com/a/17106701)

그리고 해당 라인이 파일 전체에 4번 반복되니깐 함수로 빼도 좋겠다는 생각이 드네요 👍

Reply:
오 깔끔하네요!